.gitignore

```diff
@@ -37,6 +37,7 @@ GeoIPv6.dat
 *node_modules/
 */node_modules/
 */bower_components/
```

Comment:
While you are touching this file, would you mind changing them to just `node_modules/` and `bower_components/`?
No need for these variations imo.

Reply:
@pzi I've fixed this up..